server/apiServer.js

```diff
@@ -0,0 +1,16 @@
+// Configure process.env with .env.* files
+require('./env').configureEnv();
+
+const express = require('express');
+const cookieParser = require('cookie-parser');
+const apiRouter = require('./apiRouter');
+
+const PORT = parseInt(process.env.API_SERVER_PORT, 10);
```

Comment:
I think this magic number could benefit from a comment or named variable.

Reply:
This is just the same as is done [in the main server](https://github.com/sharetribe/ftw-daily/blob/master/server/index.js#L46):

```js
const PORT = parseInt(process.env.PORT, 10);
```

Do you mean commenting on the `10` that is given to the `parseInt`? I feel that is quite usual to have it (or at least used to), and this is not really a file for customization.

Reply:
Yes, I was pointing to '10'. 
We had at some point an (eslint) rule that any number needs to have a variable. I think it makes sense to comment these somehow even if those strict eslint-rules have been removed in favour of Prettier.